Functions.Templates/Templates/SqlOutputBinding-Python/__init__.py

```diff
@@ -1,21 +1,29 @@
 import json
 import azure.functions as func
 
-def main(req: func.HttpRequest, product: func.Out[func.SqlRow]) -> func.HttpResponse:
+def main(req: func.HttpRequest, items: func.Out[func.SqlRowList]) -> func.HttpResponse:
     """Sample SQL Output Binding
 
     See https://aka.ms/sqlbindingsoutput for more information about using this binding
 
+    These tasks should be completed prior to running :
+    1. Update "commandText" in function.json - this should be the name of the table that you wish to upsert values to
+    2. Add an app setting named "SqlConnectionString" containing the connection string to use for the SQL connection
+    3. Change the bundle name in host.json to "Microsoft.Azure.Functions.ExtensionBundle.Preview" and the version to "[3.*, 4.0.0)"
+
     Arguments:
     req: The HttpRequest that triggered this function
-    product: The object to be upserted to the database
+    items: The objects to be upserted to the database
     """
 
+    # Note that this expects the body to be an array of JSON objects which
+    # have a property matching each of the columns in the table to upsert to.
     body = json.loads(req.get_body())
-    product.set(func.SqlRow.from_dict(body))
+    rows = list(map(lambda r: func.SqlRow.from_dict(r), body))
```

Comment:
There aren't any issues with this not being a SqlRowList object?

Reply:
Oh ok, looks like we have code to handle just a normal iterable thing too (as long as it contains SqlRow objects). Neato.

Reply:
Oh ok, looks like we have code to handle just a normal iterable thing too (as long as it contains SqlRow objects). Neato. I'm a little surprised that it's fine calling set on the Out object given that it's not actually that. Consider having the type be just a SqlRow array instead for that

Reply:
Yep, I tested the templates and rows were upserted correctly.

Reply:
Updated to use SqlRowList here instead.